test/e2e/apps/statefulset.go

```diff
@@ -843,6 +847,52 @@ var _ = SIGDescribe("StatefulSet", func() {
 				return nil
 			}, framework.StatefulPodTimeout, 2*time.Second).Should(BeNil())
 		})
+
+		It("should have a working scale subresource", func() {
+			By("Creating statefulset " + ssName + " in namespace " + ns)
+			ss := framework.NewStatefulSet(ssName, ns, headlessSvcName, 1, nil, nil, labels)
+			sst := framework.NewStatefulSetTester(c)
+			sst.SetHttpProbe(ss)
+			ss, err := c.AppsV1beta1().StatefulSets(ns).Create(ss)
+			Expect(err).NotTo(HaveOccurred())
+			sst.WaitForRunningAndReady(*ss.Spec.Replicas, ss)
+			ss = sst.WaitForStatus(ss)
+
+			By("posting a scale subresource")
+			scale := framework.NewStatefulSetScale(ss)
+			scaleResult := &appsV1beta2.Scale{}
+			err = c.AppsV1beta2().RESTClient().Put().AbsPath("/apis/apps/v1beta2").Namespace(ns).Resource("statefulsets").Name(ssName).SubResource("scale").Body(scale).Do().Into(scaleResult)
+			if err != nil {
+				framework.Failf("Failed to put scale subresource: %v", err)
+			}
+			Expect(scaleResult.Spec.Replicas).To(Equal(int32(1)))
+
+			By("getting scale subresource")
+
+			err = c.AppsV1beta2().RESTClient().Get().AbsPath("/apis/apps/v1beta2").Namespace(ns).Resource("statefulsets").Name(ssName).SubResource("scale").Do().Into(scale)
+			if err != nil {
+				framework.Failf("Failed to get scale subresource: %v", err)
+			}
+			fmt.Println(scale)
```

Comment:
If you need to log use framework.Log, not fmt.Println. Also, you might want to make that Log statement a little more descriptive.